src/main/java/frc/robot/drivetrains/simpleTankDrivetrain/OI.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.drivetrains.simpleTankDrivetrain;
+
+import edu.wpi.first.wpilibj.Joystick;
+
+/**
+ * Handles the communication between the user and the robot.
+ */
+public class OI /*GEVALD*/ {
+
+    private final Joystick leftJoystick = new Joystick(0);
```

Comment:
In the latest RapidReact pr, the joysticks are static. These should probably be static too.